README.md

```diff
@@ -351,3 +351,57 @@ Symbols:
 <FontAwesomeIcon icon="edit" symbol />
 <FontAwesomeIcon icon="edit" symbol="edit-icon" />
 ```
+
+### Type Script
```

Comment:
`TypeScript`?

Reply:
Ah, yes. I should have looked more closely. No space. Check.